helm/scylla/templates/scyllacluster.yaml

```diff
@@ -22,8 +22,7 @@ spec:
   {{- end }}
   {{- if .Values.alternator.enabled }}
   alternator:
-    port: {{ .Values.alternator.port }}
-    writeIsolation: {{ .Values.alternator.writeIsolation }}
+    {{- toYaml .Values.alternator | nindent 4 }}
```

Comment:
wouldn't it try to set `enabled` field, and fail due to additional property?

Reply:
fixed, thx